mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -133,20 +134,28 @@ private boolean isChangeImageRequestFromProfile() {
         return getIntent().getStringExtra(Constants.CHANGE_PROFILE_IMAGE_KEY) != null;
     }
 
+    private boolean isValidEmail(CharSequence target) {
+        return (!TextUtils.isEmpty(target) && Patterns.EMAIL_ADDRESS.matcher(target).matches());
+    }
+
```

Comment:
Remove all the extra lines, here and in the future.

Reply:
@shiv07tiwari I think instead of duplicating the above function in all the activity we can place it in org.mifos.mobilewallet.mifospay.utils.utils so that it becomes usable in whole application. What do you think?

Reply:
Sounds good.